docs/pvacfuse.rst

```diff
@@ -0,0 +1,21 @@
+.. image:: images/pVACfuse_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACfuse logo
+
+pVACfuse
+====================================
+
+This component of the pVACtools workflow provides support for predicting
+neoantigens from gene fusions. Currently, fusion variants as reported by
+`INTEGRATE-Neo <https://github.com/ChrisMaherLab/INTEGRATE-Neo>`_ are supported, and predictions against any of the
+IEDB-supported prediction softwares can be reported.
```

Comment:
s/predictions against any of the IEDB-supported prediction softwares can be reported./any of the IEDB-supported software can be used for binding prediction./